assets/js/base/components/price-slider/style.scss

```
@@ -247,12 +256,19 @@
 			}
 		}
 
-		&.is-loading {
+		&.is-loading,
+		&.is-disabled {
 			.wc-block-price-filter__range-input-wrapper {
 				@include placeholder();
 				box-shadow: none;
 			}
 		}
+
+		&.is-disabled:not(&.isLoading) {
```

Same here. :slightly_smiling_face: 
```suggestion
		&.is-disabled:not(.is-loading) {
```